eppo_core/src/lib.rs

```diff
@@ -80,5 +80,6 @@ pub use attributes::{
 pub use configuration::Configuration;
 pub use error::{Error, EvaluationError, Result};
 #[cfg(feature = "event_ingestion")]
+pub use event_ingestion::ContextValue;
 pub use sdk_key::SdkKey;
```

Comment:
Probably unintended change of API: `SdkKey` is now available without `event_ingestion` feature flag